examples/Cargo.toml

```diff
@@ -57,6 +57,8 @@ static_cell         = { version = "2.1.0", features = ["nightly"] }
 usb-device          = "0.3.2"
 usbd-serial         = "0.2.2"
 
+opentherm_boiler_controller_lib = { version = "0.1.0", path = "/home/cooler1989/projects/rusty_boiler/boiler_controller_ot_lib", features = [] }
```

Comment:
I'd like to ask you not to please do not add a dependency that lives on your filesystem.

Reply:
I am sorry - it was not intended to be public. At least not yet. I was suppose to push I lt to private fork.

Reply:
This PR, or that dependency?

Reply:
This whole PR. Can one delete it so it does not pollute this repository even in the history?